.github/workflows/issue-open.yml

```diff
@@ -0,0 +1,18 @@
+# issue-open.yml
+# mark the initial status of the issue when it's been opened
+name: Issue Open Check
```

Comment:
Can the two workflow merged into one? Or we rename the files like `reply-lable-issue-comment.yml` and `reply-lable-issue-open.yml`, Is this better?

Reply:
In a nutshell, the trigger time of two workflow is different, so we can't merged them into one workflow. As for rename the file, I think it's depends on whether it has clean meaning.